chains/evm/contracts/scripts/CCIPSendTestScript.s.sol

```diff
@@ -0,0 +1,103 @@
+pragma solidity ^0.8.24;
+
+import {Router} from "../Router.sol";
+import {Client} from "../libraries/Client.sol";
+
+import {IERC20} from "@chainlink/vendor/openzeppelin-solidity/v5.0.2/contracts/token/ERC20/IERC20.sol";
+import {SafeERC20} from "@chainlink/vendor/openzeppelin-solidity/v5.0.2/contracts/token/ERC20/utils/SafeERC20.sol";
+
+import {Script} from "forge-std/Script.sol";
+
+/* solhint-disable-next-line no-console */
+import {console2 as console} from "forge-std/console2.sol";
+
+/* solhint-disable no-console */
+contract CCIPSendTestScript is Script {
+  using SafeERC20 for IERC20;
+
+  address public ROUTER;
+  address public FEE_TOKEN;
+
+  address public TOKEN0;
+  uint256 public TOKEN0_AMOUNT;
+
+  uint64 public DESTINATION_CHAIN_SELECTOR;
+  uint64 public SOURCE_CHAIN_SELECTOR;
+
+  // Ex: "ETHEREUM_RPC_URL" as defined in .env
+  string public RPC_DESCRIPTOR;
+
+  bytes public s_extraArgs;
+  bytes public s_recipient;
+  bytes public s_data;
+
+  function run() public {
+    vm.createSelectFork(RPC_DESCRIPTOR);
+
+    uint256 privateKey = vm.envUint("PRIVATE_KEY");
+
+    address sender = vm.rememberKey(privateKey);
+    s_recipient = abi.encode(sender);
```

Comment:
Why store this in storage? Why do we override the values that appeared to be the input params (storage vars)